### PR TITLE
chore: Switch to using nuxt icon module

### LINKED ISSUE #4251: Switch to using nuxt icons module ( https://nuxt.com/modules/icon)
**Is your feature request related to a problem? Please describe.**
Icons support ( on page, with buttons, sizes, colours, animation)

**Describe the solution you'd like**
Switch to using https://nuxt.com/modules/icon ( instead of custom solution)

**Describe alternatives you've considered**
We currently have a custom solution, down side of custom solution is we need to maintain the tool chain and exposing icons as global components is less than ideal

### PR DESCRIPTION
**Pull request by connoratrug**
closes https://github.com/molgenis/molgenis-emx2/issues/4251

What are the main changes you did:
- setup module in tailwind-components lib
- add molgenis icons to the setup
- add samplepage to view/text icons in playground
- 'fix' the svg's to use current fill
- remove 'old' icons and tooling
- rename custom icon files to follow nuxt/icon name schema ( kebab-case )
- open up icon route via ingress ( for k8, vm's need update of nginx config ⚠️ )
- setup app config for playground, catalogue and ui apps 

- [ ] test local
- [ ] test non ssr playground setup 
- [ ] test ssr app usage setup ( catalogue )

### CODE REVIEW

**Review by mswertz — COMMENTED** [review]
looks good, but I am trying to understand the exact benefits. I see that this library gives accesss to 200,000+ icons, that is great. Will we stop generating the icons? Any other benefits I should understand?

File: apps/tailwind-components/pages/Button.story.vue [hunk 1]
```diff
@@ -42,19 +42,34 @@
...
+        icon="material-symbols:kid-star-outline-sharp"
+        iconPosition="left"
       >
+        Star
+      </Button>
       <Button
         type="secondary"
         size="small"
-        icon="CaretDown"
+        icon="line-md:chevron-small-down"
```
[the remark this answers is not in the document]
Comment (mswertz) [reply apps/tailwind-components/pages/Button.story.vue:72]: there is not, it is growing on me. It is just very long names but if we would care very much we could make a mapping list. I don't care for now. Better stick to something documented elsewhere

**Review by davidruvolo51 — CHANGES_REQUESTED** [review]
This is really great! I noticed one issue in the story and I had a few comments. Otherwise, everything looks great!

File: apps/tailwind-components/README.md [hunk 2]
```diff
@@ -18,33 +18,8 @@ In the component folder, we've arranged the components into categories. To use t
...
-
-Before adding a new icon to the component library, make sure there isn't a similar icon in `/global/icons` folder. If an appropriate icon isn't available, download the svg from [https://fonts.google.com/icons](https://fonts.google.com/icons) and [https://www.flaticon.com/authors/special/lineal](https://www.flaticon.com/authors/special/lineal). Place the file in the `assets/icons` folder and run the parse icons command.
-
-```bash
-yarn parse-icons
-```
-
-This script uses the [sgvo](https://github.com/svg/svgo) module to clean the icons and transform them into vue components. These files are then saved in the `global/icons/` folder.
+Icons are rendered using @nuxt/icon, for usage details and options see: https://nuxt.com/modules/icon
+The custom (default) icons used in the component library are stored in the assets/icons folder
```
Comment (mswertz) [inline apps/tailwind-components/README.md:22]: anything special one needs to do to these icons?

**Review by mswertz — COMMENTED** [review]
Does this close the door for non-nuxt based use of tailwind components?
(or was that door closed already?)

File: apps/tailwind-components/assets/icons/completed.svg
Comment (davidruvolo51) [inline apps/tailwind-components/assets/icons/completed.svg:1]: Wow! It surprises me that the dimensions are so large.

**Review by davidruvolo51 — APPROVED** [review]

**Pull request closed without merging**